Source/v3Net/MeetupBot/MeetupBot.csproj

```diff
@@ -284,7 +300,11 @@
     <Content Include="Helpers\AdaptiveCards\PairUpNotificationAdaptiveCardPerson2.json" />
     <Content Include="Helpers\AdaptiveCards\WelcomeNewMemberCard.json" />
     <Content Include="Helpers\AdaptiveCards\PairUpNotificationAdaptiveCardPerson1.json" />
+    <Content Include="Helpers\AdaptiveCards\WelcomeNewMemberCardDev.json" />
```

Comment:
Can you add these files also to the repo

Reply:
Fixed.